staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,55 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, this directory will be skipped.
+
+- Adding `.import-restrictions` files does not add them to CI runs. They need to be explicitly added to `hack/verify-import-boss.sh`.
```

Comment:
is this true? cc @eloyekunle

Reply:
Once a parent directory has been added to `hack/verify-import-boss.sh`, the subpackages do not need to be added but will be checked automatically.

Currently, everything in `staging/src/k8s.io/*/`, as well as `k8s.io/kubernetes/pkg/..., k8s.io/kubernetes/cmd/..., k8s.io/kubernetes/plugin/..., k8s.io/kubernetes/test/e2e/framework/...` are currently checked, and that includes most of the code in k/k.